NEWS.md

```diff
@@ -1,4 +1,26 @@
-## 5.7.0 - Unreleased
+## 5.8.0 - Unreleased
+
+## 5.7.0 - Released (Poppy R2 2023)
+
+### Breaking changes
+* Description ([MODINVOICE-466](https://issues.folio.org/browse/MODINVOICE-466))
+
+### New APIs versions
+* Requires `PUT v5.7.0` extra fiscalYearId in DTO
+
+### Features
+* Update to Java 17 mod-invoice-storage ([MODINVOSTO-166](https://issues.folio.org/browse/MODINVOSTO-166))
+* Create invoice line numbers based on new invoice field ([MODINVOSTO-162](https://issues.folio.org/browse/MODINVOSTO-162))
+* Update models schema with new fields for ftp configuration ([MODINVOICE-455](https://issues.folio.org/browse/MODINVOICE-455))
+* Update dependent raml-util ([MODINVOSTO-154](https://issues.folio.org/browse/MODINVOSTO-154))
+* Update DTO schema with adding fiscalYearId to PUT payload ([MODINVOSTO-144](https://issues.folio.org/browse/MODINVOSTO-144))
```

Comment:
Please follow release notes as it was

Reply:
I read this documentation https://wiki.folio.org/pages/viewpage.action?pageId=96404780. Looks like there is no documentation to follow common documentation for across all teams. 

I will try to follow existed prs.